packages/schematics/src/collection/application/files/__directory__/tslint.json

```diff
@@ -8,6 +8,9 @@
     "callable-types": true,
     "class-name": true,
     "forin": true,
+    "deprecation": {
+      "severity": "warn"
```

Comment:
Why did you need this?

Reply:
This is to match what CLI 1.6.5 generates by default.